connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorCheckpointConnector.java

```diff
@@ -106,6 +108,17 @@ public void stop() {
         Utils.closeQuietly(targetAdminClient, "target admin client");
     }
 
+    @Override
+    public Config validate(Map<String, String> connectorConfigs) {
+        List<ConfigValue> configValues = super.validate(connectorConfigs).configValues();
+        MirrorCheckpointConfig.validate(connectorConfigs).forEach((config, errorMsg) ->
+                configValues.stream()
+                        .filter(conf -> conf.name().equals(config))
+                        .forEach(conf -> conf.errorMessages().add(errorMsg)));
```

Comment:
could you please use `addErrorMessage` instead of `errorMessages().add`?

Reply:
updated this. Forgot to roll it back to addErrorMessage when addressing some of other feedbacks